src/channels/views/ChannelDetails/ChannelDetails.tsx

```diff
@@ -278,10 +284,10 @@ export const ChannelDetails: React.FC<ChannelDetailsProps> = ({
         countries={shop?.countries || []}
       />
       <ChannelDeleteDialog
-        channelSlug={data?.channel?.slug}
-        currency={data?.channel?.currencyCode}
+        channelSlug={data?.channel?.slug as string}
+        currency={data?.channel?.currencyCode as string}
```

Comment:
suggestion: casting value that could possible be undefined to string it is not save IMO
```suggestion
        channelSlug={data?.channel?.slug  || ''}
        currency={data?.channel?.currencyCode || ''}
 ```